src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -193,6 +216,13 @@ internal void RefreshLibraryView(WebView2 browser)
 
         #endregion
 
+        internal void OnCopyToClipboard(string text)
+        {
+            Clipboard.SetText(text);
+        }
+
+        internal void OnPasteFromClipboard() { }
```

Comment:
This method does nothing for now I assume, please add comments mentioning this is a temparary state